### ISSUE TITLE
Symbolic package not working because of SymPy errors

### ISSUE THREAD

**Issue by adamesalles** [cbm755/octsympy#1130]
I'm moving from Matlab to Octave and when I first tried to install the symbolic package and use it via `pkg`, my first attempts with `sym` are not working.

```octave
>> sym(A)
Symbolic pkg v2.9.0: Traceback (most recent call last):
  File "<stdin>", line 28, in <module>
AttributeError: '_PrintFunction' object has no attribute '__globals__'
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<stdin>", line 12, in octoutput_drv
  File "<stdin>", line 54, in octoutput
  File "<stdin>", line 55, in octoutput
AttributeError: module 'sympy' has no attribute 'compatibility'
Closing the Python communications link.

error: Python exception: AttributeError: '_PrintFunction' object has no attribute '__globals__'
    occurred in python_header import block.
    Try "sympref reset" and repeat your command?
    (consider filing an issue at https://github.com/cbm755/octsympy/issues)
error: called from
    pycall_sympy__ at line 191 column 5
    double_to_sym_heuristic at line 46 column 7
    sym at line 379 column 13
    numeric_array_to_sym at line 36 column 16
    sym at line 359 column 7
```

I ran `sympref diagnose` and that's the output:
```octave
>> sympref diagnose

Symbolic package diagnostics
============================

Python and SymPy are needed for most features of the Symbolic package.

The Python interpreter is currently: "python".

Computers may have more than one Python interpreter installed.  If you
need to, you can select a different one using the PYTHON environment
variable (see "help sympref").  For example, to use Python 2, try
    setenv PYTHON python2
    sympref reset

Attempting to run python -c "print(\"Python says hello\")"

status = 0
output = Python says hello

Good, Python ran correctly.


Python version
--------------

Let's check what version of Python we are calling...

Attempting to run python -c "import sys; print(sys.version)"

status = 0
output = 3.10.4 (main, Mar 23 2022, 23:05:40) [GCC 11.2.0]


SymPy Python Library
--------------------

SymPy is a Python library used by Symbolic for almost all features.

Attempting to run python -c "import sympy; print(sympy.__version__)"

status = 0
output = 1.10.1

SymPy must be at least version 1.2; you have version 1.10.1.

Good, a working version of SymPy is installed.


Python XML Parsing and DOM Support
----------------------------------

The XML DOM library is used by Symbolic for passing values to and from Python.
Some older versions of Python formatted XML output differently.  As long as you
have any reasonably recent version of Python, this should pass.

Attempting to run python -c "import xml.dom.minidom as minidom; doc = minidom.parseString(\"<item>value</item>\"); print(doc.toprettyxml(indent=\"\"))"

status = 0
output = <?xml version="1.0" ?>
<item>value</item>



Your kit looks good for running the Symbolic package.  Happy hacking!
```

I've used SymPy in the past, and I run Arch Linux.

**Comment by AnotherUser12** [comment]
AHHHHH

**Comment by cbm755** [comment]
Thanks, we know and are working on a 3.0.0 release to fix this and other problems

**Comment by cbm755** [comment]
I think this is all fixed in main, and upcoming 3.0.0
